### ISSUE TITLE
Incremental passthrough copy

### ISSUE THREAD

**Issue by zachleat** [11ty/eleventy#977]
When using `--watch` or `--serve`, passthrough copy files shouldn’t require a full build when using `--incremental` or on an opt-in basis individually configured in your Configuration file.

They should be able to copy while a build is ongoing without interrupting a build or triggering another build after the current one finishes.

**Comment by zachleat** [comment]
> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by zachleat** [comment]
Two pieces of this, allowing passthrough copy files to be processed separately/incrementally and allowing parallel builds so that copy can happen _while_ another build is running. Going to punt parallel builds to a separate issue here, see https://github.com/11ty/eleventy/issues/1001

**Comment by zachleat** [comment]
Also going to wait on Configuration API for individual entries until someone else asks for it 😅

**Comment by marcfilleul** [comment]
So with Eleventy 0.11, is the incremental passthrough copy automatic or do I need to add a — incremental flag?

**Comment by zachleat** [comment]
@marcfilleul Incremental features require `--incremental` for now. More discussion about this at #1109

**Comment by marcfilleul** [comment]
Thank you Zach, it's a game changer ;)
